java/org/apache/catalina/manager/Constants.java

```diff
@@ -39,38 +39,65 @@
             "<head>\n" +
             "<style>\n" +
             org.apache.catalina.util.TomcatCSS.TOMCAT_CSS + "\n" +
+            "  #manager-wrapper {\n" +
+            "    min-width: 720px;\n" +
+            "    max-width: 1000px;\n" +
+            "    margin: 20px auto;\n" + 
+            "    background: #fff;\n" + 
+            "    padding: 20px 30px;\n" +
+            "  }\n" +
             "  table {\n" +
             "    width: 100%;\n" +
+            "    margin: 10px auto 0px auto;\n" +
             "  }\n" +
             "  td.page-title {\n" +
+            "    font-size:52px; \n" +
             "    text-align: center;\n" +
             "    vertical-align: top;\n" +
             "    font-family:sans-serif,Tahoma,Arial;\n" +
             "    font-weight: bold;\n" +
             "    background: white;\n" +
+            "    text-align: left;\n" +
+            "    padding-top:20px\n" +
             "    color: black;\n" +
+            "    padding-top: 24px;\n" +
             "  }\n" +
+            "  td.page-title font{\n" +
+            "    font-size:52px;\n" +
+            "  }\n" + 
+            "  #manager-section{\n" +
+            "    margin-bottom:20px;\n" +
+            "  }\n" +   
+            "  #server-section{\n" +
+            "    margin-top:20px;\n" +
+            "    margin-bottom:170px;\n" +
+            "  }\n" +  
+            "  #copyright{\n" +
+            "    display:block;\n" +
+            "    margin-bottom:30px;\n" +
+            "  }\n" +               
             "  td.title {\n" +
+            "    color: #fff;\n" + 
             "    text-align: left;\n" +
             "    vertical-align: top;\n" +
             "    font-family:sans-serif,Tahoma,Arial;\n" +
-            "    font-style:italic;\n" +
+            "    font-size:23px;\n" +
```

Comment:
Try to use relative sizes and not explicit pixel sizes. It will allow the UI to scale much more nicely on a variety of devices.